src/tomlToJSON.cc

```diff
@@ -21,6 +21,29 @@ namespace flox {
 
 /* -------------------------------------------------------------------------- */
 
+/** @brief An exception thrown when converting TOML to JSON */
+struct TOMLToJSONException : public FloxException {
+  static constexpr std::string_view categoryMsg = "error converting TOML to JSON";
```

Comment:
Are we only ever converting TOML to JSON when constructing a registry? If so that may be a more useful error message.

Reply:
I don't know